.github/workflows/build-test.yml

```diff
@@ -32,7 +32,10 @@ jobs:
     - name: Install dependencies
       run: |
         pip install .[docs]
-        pip install pytest black isort pytest-cov
+        pip install pytest
+        pip install black=="22.6.0"
+        pip install isort=="5.10.1"
+        pip install pytest-cov
```

Comment:
```suggestion
        pip install pytest black=="22.6.0" isort=="5.10.1" pytest-cov
```

we don't need multiple `pip install` commands, it would be slower.